### ISSUE TITLE
[Accessibility] keyboard-only users cannot drag tree separator via keyboard.

### ISSUE THREAD

**Issue by setaskin** [microsoft/vscode#120261]
- VS Code Version: 1.54.3 stable 
- OS Version: Windows 

Steps to Reproduce:

1. click on the separator in between tree view and the editor.
expected : all users should be able to drag the separator either via mouse or keyboard
actual :  keyboard-only users unable to drag the separator via keyboard. 

![image](https://user-images.githubusercontent.com/37163639/113188627-20f4a200-920f-11eb-98a7-f035e19ee672.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by setaskin** [comment]
@isidorn , this is another bug accessibility team assigned to Live Share, I believe root cause coming from VS Code repo.

**Comment by joaomoreno** [comment]
@isidorn I'd like to know how other products achieve this. What's a keyboard accessible way to move sashes? Is there such a concept even? Should we have shortcuts to increase it and decrease it a fraction of the total size, say `10%` of total viewport?

**Comment by isidorn** [comment]
We already have this
`F1 > Increase current view size`
`F1 > decrease current view size`

These commands can be assigned keyboard shortcuts to by the user.
Thus closing as already working
